src/main/java/com/tstepnik/planner/domain/Task.java

```diff
@@ -13,23 +13,18 @@
     @NotEmpty
     private String description;
 
+    @Enumerated(EnumType.STRING)
+    private Importance importance;
+
     private boolean isDone;
-    private boolean isArchived;
+
 
     @ManyToOne
     private User user;
```

Comment:
I left you a comment about this field in the old PR. Should we store User or just his id?

Reply:
In database to task table is added only user Id.

Reply:
I know what is added to database, but as you can see because of having user in Task introduces a bunch of issues. Did you read my comments on the old pr?

Reply:
Yes I did but I still don't know how to make databases conected only by user_id.